apps/docs/src/app/core/component-docs/dialog/examples/dialog-backdrop-container/dialog-backdrop-container-example.component.html

```diff
@@ -18,13 +18,11 @@
     <fd-dialog [dialogRef]="dialog" [dialogConfig]="dialogConfig">
         <fd-dialog-header>
             <h1 id="fd-dialog-header-2" fd-title>Backdrop Configuration</h1>
-            <button fd-dialog-close-button (click)="dialog.dismiss()"></button>
+            <button fd-dialog-close-button (click)="dialog.dismiss()" i18n-title="@@dialogPlayGround.closebutton.title"></button>
```

Comment:
`i18n-title` not needed here as explained above. `title` is needed.

Reply:
fixed